src/main/kotlin/no/nav/helse/rapids_rivers/Metrics.kt

```diff
@@ -7,16 +7,22 @@ import io.prometheus.client.Histogram
 object Metrics {
     private val registry = CollectorRegistry.defaultRegistry
 
+    val lagHistogram = Histogram.build()
+        .name("rapids_rivers_lag_seconds")
+        .help("Lag mellom sending av melding og mottak av melding")
```

Comment:
Eventuent "Lag" -> "Consumer lag" for tydelighet, men det koster nok ikke så mye å skjønne seg fram til hva det er snakk om.